### PR TITLE
Improve docs of bitwarden lookup

### LINKED ISSUE #7718: Get clean data from bitwarden
### Summary

Unless I am missing something, I think the ansible bitwarden module is useless: I can make ansible it retrieve passwords from my bitwarden vault and inject them in my desired config files, but it retrieves a json object... which means that when using this to fill in passwords it will use the format **['password']** instead of the **plain** password....  is there a way to tell the ansible module to ONLY get the password field content and not put it in a stupid json?

Thanks and regards

### Issue Type

Feature Idea

### Component Name

bitwarden

### Additional Information

```yaml (paste below)

```

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Fixes #7718.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
bitwarden lookup

### CODE REVIEW

File: plugins/lookup/bitwarden.py [hunk 1]
```diff
@@ -25,7 +25,10 @@
         type: list
         elements: str
       search:
-        description: Field to retrieve, for example V(name) or V(id).
+        description:
+          - Field to retrieve, for example V(name) or V(id).
+          - If set to V(id), only zero or one element can be returned.
+            Use the Jinja C(first) filter to get hold of the only list element.
```
Comment (oraNod) [inline plugins/lookup/bitwarden.py:31]:
```suggestion
            Use the Jinja C(first) filter to get the only list element.
```

File: plugins/lookup/bitwarden.py [hunk 2]
```diff
@@ -39,37 +42,40 @@
...
   ansible.builtin.debug:
     msg: >-
       {{ lookup('community.general.bitwarden', 'a_test', field='api_key') }}
 """
 
 RETURN = """
   _raw:
-    description: List of requested field or JSON object of list of matches.
+    description:
+      - A one-element list of a list of requested fields or JSON objects of matches.
```
Comment (oraNod) [inline plugins/lookup/bitwarden.py:74]: is it a list of lists? or can we say "a list of requested fields" without changing meaning?

File: plugins/lookup/bitwarden.py [hunk 3]
```diff
@@ -39,37 +42,40 @@
...
+      - If you use C(query), you always get a list of lists. If you use C(lookup) without C(wantlist=true),
```
Comment (oraNod) [inline plugins/lookup/bitwarden.py:75]:
```suggestion
      - If you use C(query), you get a list of lists. If you use C(lookup) without C(wantlist=true),
```

Comment (oraNod) [inline plugins/lookup/bitwarden.py:75]:
```suggestion
      - If you use C(query), you get a list of lists. If you use C(lookup) without C(wantlist=true),
```

File: plugins/lookup/bitwarden.py (hunk 2 above)
Comment (oraNod) [reply plugins/lookup/bitwarden.py:74]: I think the descriptions after this answer my question...

Comment (felixfontein) [reply plugins/lookup/bitwarden.py:74]: > is it a list of lists?

Unfortunately yes. It is a one-element list, whose only element is a list which contains the actual fields.

Comment (felixfontein) [reply plugins/lookup/bitwarden.py:74]:
```suggestion
      - A one-element list that contains a list of requested fields or JSON objects of matches.
```

**Comment by felixfontein** [comment]
@oraNod thanks for reviewing and improving this!

**Pull request merged**